From 59395730bc66d97c154f727543e501d6f3c061af Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd@kuhls.net>
Date: Sat, 25 Nov 2023 14:03:45 +0100
Subject: [PATCH] Fix build with libxml2-2.12

Upstream reorganized header includes:
https://github.com/GNOME/libxml2/blob/master/NEWS

"Several cyclic dependencies in public header files were fixed. As a
 result, certain headers won't include other headers as before."

Upstream: https://sourceforge.net/p/xmlstar/patches/22/

Signed-off-by: Bernd Kuhls <bernd@kuhls.net>
---
 src/xmlstar.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/xmlstar.h b/src/xmlstar.h
index 3e1eed3..1a12da1 100644
--- a/src/xmlstar.h
+++ b/src/xmlstar.h
@@ -15,6 +15,7 @@
 #include <libxml/xpath.h>
 #include <libxml/xpathInternals.h>
 #include <libxml/xmlreader.h>
+#include <libxml/xmlsave.h>
 
 typedef enum { /* EXIT_SUCCESS = 0, EXIT_FAILURE = 1, */
     EXIT_BAD_ARGS = EXIT_FAILURE+1, EXIT_BAD_FILE,
-- 
2.39.2

